PSKoans/Data/Advice/Customization/profile.txt

```diff
@@ -0,0 +1,6 @@
+  PowerShell Profile:
+When starting powershell, there is a script it will automatically execute:
+The PowerShell profile script
+By default, it does not exist. It is stored in the location the variable $profile points to. By creating and adding code to it you can ensure that all your favorite aliases, variables, functions or whatnot will automatically be available each time!
+To create or open the profile file, simply run:
+  notepad $profile
```

Comment:
Don't think we should mention multiple profiles here: Adds several lines and muddles the water.
I'd rather make that a second advice. Which kind of introduces the issue of advice dependencies - one only showing if another has been displayed. Meaning we'd need to track what we displayed.

Reply:
Agreed that it should be a separate snippet.

Not sure we should worry too much about 'dependencies' per se... but it might be worth looking into. 😄